packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -597,6 +634,149 @@ export const prepareVaultManagerKit = (
           return E(metricsTopicKit.recorder).write(payload);
         },
 
+        /**
+         * @param {TimestampRecord} timestamp
+         * @returns {Promise<LiquidationVisibilityWriters>}
+         */
+        async makeLiquidationVisibilityWriters(timestamp) {
+          const liquidationRecorderKits =
+            await this.facets.helper.makeLiquidationRecorderKits(timestamp);
```

Comment:
We prefer a local declaration of `helper` rather than destructuring in-line.
```suggestion
          const { facets: { helper } } = this;
          const liquidationRecorderKits =
            await helper.makeLiquidationRecorderKits(timestamp);
```

Reply:
The fix will be included in the next iteration.